github/Workflow.py

```diff
@@ -137,13 +137,15 @@ def get_runs(
         branch=github.GithubObject.NotSet,
         event=github.GithubObject.NotSet,
         status=github.GithubObject.NotSet,
+        head_sha=github.GithubObject.NotSet,
     ):
         """
         :calls: `GET /repos/{owner}/{repo}/actions/workflows/{workflow_id}/runs <https://docs.github.com/en/rest/reference/actions#workflow-runs>`_
```

Comment:
Please update the docs link to https://docs.github.com/en/rest/actions/workflow-runs#list-workflow-runs-for-a-workflow